src/hooks/resources/device.ts

```diff
@@ -579,3 +595,105 @@ async function checkSupervisorReleaseUpgrades(
 		}
 	}
 }
+
+async function setSupervisorReleaseResource(
+	api: sbvrUtils.PinejsClient,
+	deviceIds: number[],
+	supervisorVersion: string,
+) {
+	if (deviceIds.length === 0) {
+		return;
+	}
+	const devices = (await api.get({
+		resource: 'device',
+		options: {
+			$expand: {
+				is_of__device_type: {
+					$select: 'id',
+				},
+			},
+			// if the device already has a supervisor_version, just bail.
+			$filter: {
+				id: { $in: deviceIds },
+				supervisor_version: { $eq: null },
```

Comment:
`$eq` is the default so you can simplify this to
```suggestion
				supervisor_version: null,
```